builder/src/webpack.config.ext.ts

```diff
@@ -191,13 +217,14 @@ module.exports = [
           type: 'var',
           name: ['_JUPYTERLAB', data.name]
         },
-        filename: 'remoteEntry.js',
+        filename: 'remoteEntry.[contenthash].js',
         exposes,
         shared
-      })
+      }),
+      new CleanupPlugin()
     ]
   })
 ].concat(extras);
 
-const logPath = path.join(outputPath, 'build_log.json');
-fs.writeFileSync(logPath, JSON.stringify(module.exports, null, '  '));
+// const logPath = path.join(outputPath, 'build_log.json');
```

Comment:
Why remove the log?  I've used several times.

Reply:
I'll put in another PR that only saves it in development mode. Does that sound good to you?

Reply:
Sounds good.  There is a conflict here, I'm afk for the next couple hours but I can rebase and release then.